site/app/templates/EditDisplayNameOrderForm.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}edit-display-name-order-form{% endblock %}
+{% block title %}Edit Display Name Order{% endblock %}
+{% block body %}
+    <p class="std-margin">{{ "You can edit the order in which your name is displayed here!" }}</p>
+    <div class="flex-col" id="user_settings_modal">
+        <input id="display-name-order-csrf" type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+        <h2>My Pronouns</h2>
+        <div class="edit_user_row ui-front">
+    <select name="display-name-order-change" id="display-name-order-change">
+        {% if user.getDisplayNameOrder() != 'REVERSED_F' %}
+        <option value="GIVEN_F">Given Name First</option>
+        <option value="REVERSED_F">Family Name First</option>
+        {% endif %}
+        {% if user.getDisplayNameOrder() != 'GIVEN_F' %}
+        <option value="REVERSED_F">Family Name First</option>
+        <option value="GIVEN_F">Given Name First</option>
+        {% endif %}
+    </select>
```

Comment:
Option values have a 'selected' parameter that you can add, like `<option value="GIVEN_F" {user.getDisplayNameOrder() == 'GIVEN_F' ? selected : ''>Family Name First</option>`, so the order can be consistent instead of switching the order depending on what is selected. 

Also, these variable names aren't intuitive, since from the displayed values, I think that `GIVEN_FIRST` and `FAMILY_FIRST` or something similar make more sense (if eventually we want to add reversing, so given name last, `REVERSED` would be confusing, as it isn't reversed.

Reply:
Can't change `GIVEN_F` to anything else for database reasons, so changed `REVERSED_F` to `FAMILY_F`.